api/desecapi/views.py

```diff
@@ -198,19 +186,18 @@ def create(self, request, *args, **kwargs):
         if self.kwargs.get('subname') is not None or self.kwargs.get('type') is not None:
             raise MethodNotAllowed(method=request.method)
 
+        type = request.data.get('type', '').upper()
```

Comment:
Should we implement this as a Django permission? The `DoNotQuerySOARecordPermission` or something?